packages/booter-lb3app/src/lb3app.booter.ts

```diff
@@ -5,6 +5,7 @@
 
 import {BootBindings, Booter} from '@loopback/boot';
 import {CoreBindings, inject} from '@loopback/core';
+import {AnyObject, DataSource} from '@loopback/repository';
```

Comment:
You cannot import from `@loopback/repository` because it is not a dependency of `@loopback/booter-lb3app`, and IMO it should not be. Let's remove this import and use different types, see my next comment.